Makefile

```diff
@@ -159,6 +159,9 @@ distclean: go.distclean clean
 prune:
 	@$(MAKE) -C images prune
 
+csv:
+	@cluster/olm/ceph/generate-rook-csv.sh $(CSV_VERSION) $(CSV_PLATORM) $(ROOK_OP_VERSION)
```

Comment:
Should this var be `CSV_PLATFORM`?

Reply:
Other storage providers would be able to use this script too? Or perhaps we end up with `csv-ceph`, `csv-edgefs`, etc. We can worry about that in a future PR.

Reply:
For now, the script only works for Ceph so I'm changing this to csv-ceph.